website/content/docs/enterprise/consistency.mdx

```diff
@@ -192,6 +192,20 @@ The option `when_inconsistent` controls how stale reads are prevented:
   `X-Vault-Inconsistent: forward-active-node` header as described above under
   Conditional Forwarding
 
+## Vault 1.10 Mitigations
+
+In Vault 1.10, the token format has changed. Service tokens employ 
+server side consistency. This means that by default, requests made 
+to nodes which cannot support read-after-write consistency due to 
+not having the necessary WAL index to check vault tokens locally will yield 
+a 412 status code. The vault go api automatically retries when receiving 412s, so 
+unless there is a considerable replication delay, end-users will experience
+read-after-write consistency. 
+
+The replication option [allow_forwarding_via_token](/docs/configuration/replication)
+can be used to enforce that requests that would have returned 412s in the
```

Comment:
```suggestion
can be used to enforce requests that would have returned 412s in the
```

Reply:
I'm figuring out to see if I can make this sentence a bit clearer. 

The replication option [allow_forwarding_via_token](/docs/configuration/replication)
can be used to enforce requests that would have returned 412s in the aforementioned way will be forwarded instead to the active node.